core/imgread/common.cpp

```diff
@@ -211,11 +202,11 @@ bool InitDrive(u32 fileflags)
 	if (!InitDrive_(fn))
```

Comment:
Why not rewrite this as `if (InitDrive(fn)) return true; else { /* no disk handling code*/; return false; }`? I think it would make the code easier to read.

Reply:
No need for an else if we return from the true case. This would result in even less nesting.